guides/how_to_use_git.md

```diff
@@ -0,0 +1,24 @@
+### Overview
+
+In this tutorial, you will learn step-by-step of how to use git as follow:
+
+- How to create an account for git
+- How to create a repo on git
```

Comment:
Similarly, I think you mean "on github"? Maybe we should present how to create & manage a repo locally first, then introduce github once we move to sharing repo, collaborating, etc.?

Reply:
Solved.